### PR TITLE
Fix Humdrum to MEI conversion of arpeggios to put all the arpeggiated…

### PR DESCRIPTION
**Pull request by gregchapman-dev**
… notes/chords in arpeg@plist.

Also a few other random fixes I have written issues about.

### CODE REVIEW

**Comment by gregchapman-dev** [comment]
Issues addressed: #3116 #3119 #3121 #3148, plus the Xcode project fix that was recently committed to the develop branch.

File: src/iohumdrum.cpp [hunk 1]
```diff
@@ -11593,6 +11597,7 @@ template <class ELEMENT> void HumdrumInput::addArticulations(ELEMENT element, hu
         else if ((ch == '\'') && (posch == '\'')) {
             // staccatissimo alternate (eventually remove)
             ch = '`';
+			intch = (unsigned char)ch;
```
Comment (ahankinson) [inline src/iohumdrum.cpp:11600]: Something weird on the indents here?

**Comment by gregchapman-dev** [comment]
I’ll fix; tabs vs spaces in my editor or some such thing.

File: src/iohumdrum.cpp [hunk 2]
```diff
@@ -22957,33 +22956,32 @@ void HumdrumInput::addArpeggio(Object *object, hum::HTp token)
...
-        }
-        else {
-            secondid = getLocationId("note", earp);
-        }
-        // std::string plist = "#" + firstid + " #" + secondid;
         arpeg->AddRef("#" + firstid);
-        arpeg->AddRef("#" + secondid);
+
+        for (auto it = arpTokens.begin() ; it != arpTokens.end(); ++it) {
+			hum::HTp earp = *it;
```
[the remark this answers is not in the document]
Comment (gregchapman-dev) [reply src/iohumdrum.cpp]: Fixed

**Comment by craigsapp** [comment]
#3116, #3119 and #3148 are converting to MEI data as expected.   #3121 is not yet converting as expected with the minimal test example.

> I’ll fix; tabs vs spaces in my editor or some such thing.

Related to this, I use `clang-format -i src/iohumdrum.cpp` just before committing to adjust the formatting to make it compliant with verovio style.

(I am using version 14.0.6 of clang-format).

**Comment by craigsapp** [comment]
Related to `clang-format` here were a couple of placed that were stylistically fixed when @rettinghaus ran it:

https://github.com/rism-digital/verovio/pull/3155/files#diff-97754affb772a7d675e76486844b3a2732a4f8405eb8c2f683eb0fee11cac2e6

Screen Shot 2022-12-08 at 11 57 40 AM

And there were some tabs in iohumdrum.h:

Screen Shot 2022-12-08 at 11 58 33 AM

**Comment by gregchapman-dev** [comment]
Installing clang-format now... 🤦🏻‍♂️

**Comment by lpugin** [comment]
Great. See https://book.verovio.org/contributing/guidelines.html#formatting (we just updated to 15, I think that is the one Hombrew gives)

**Pull request merged**